locale/en/foreman_azure_rm.po

```diff
@@ -0,0 +1,171 @@
+# English translations for foreman_azure_rm package.
+# Copyright (C) 2020 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the foreman_azure_rm package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: foreman_azure_rm 1.0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: 2020-04-08 06:57-0400\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```

Comment:
Not much aware though, yet I just wanted to know if the correct name and email id is required here?

Reply:
no its not required, actually when we will upload the .pot file on transifex it will be updated with required details,

https://github.com/theforeman/foreman_virt_who_configure/blob/master/locale/en/foreman_virt_who_configure.po

I am not sure if we need any translations for english(en) though as it should be treated as source language.